app/components/layout/FindHeader.scss

```diff
@@ -15,25 +15,67 @@
     padding: calc-em(20px) 0 calc-em(65px);
   }
 
-  h1 {
-    @extend %font-size-hero;
-    @extend %base-max-width;
-    font-weight: 600;
-    color: $color-grey8;
-    text-align: center;
-    margin-bottom: calc-em(12px);
+  .header-container {
+    @extend %search-max-width;
+    display: flex;
     @include r_max($break-tablet-s) {
       width: 100%;
-      margin-bottom: calc-em(18px);
-      max-width: 20rem;
+      padding: 0 calc-em(20px) 0 calc-em(20px);
+    }
+
+    .header-text {
+      width: 70%;
+      @include r_max($break-tablet-s) {
+        width: 100%;
+      }
+
+      p {
+        color: $color-grey5;
+      }
+
+      h1 {
+        @extend %font-size-hero;
+        font-weight: 600;
+        color: $color-grey8;
+        @include r_max($break-tablet-s) {
+          width: 100%;
+          margin-top: calc-em(13px);
+          max-width: 20rem;
+        }
+      }
+    }
+
+    .location-pins {
+      width: 50%;
+      position: relative;
+      @include r_max($break-tablet-s) {
+        display: none;
+      }
+
+      .location-pin {
+        background-repeat: no-repeat;
+        background-size: contain;
+        background-position: center;
+        width: calc-em(42px);
+        height: calc-em(56px);
+        position: absolute;
+        &.pin-1 {
+          background-image: url(../../assets/img/location-pin-1.png);
+          top: calc-em(35px);
```

Comment:
What are these `calc-em`s for? I have never seen them and do not know their benefit over just using `em` or what not. Especially since you are calculating them off of a fixed `px` value, wouldn't it just be better to set them to `35px`?

Reply:
I don't know exactly why we have `calc-em()`, but I think it's probably not necessary anymore given that [`rem`](https://developer.mozilla.org/en-US/docs/Web/CSS/length) is supported by every browser including IE 9 and up. `em` is better than `px` in cases where you want measurements to be proportional to the font size (for accessibility reasons, some users may configure their browser to use a larger base font size), but `rem` is better than `em` in most cases where you want a measurement to be proportional to the root font size and not the font size of a specific element. It looks like [`calc-em()`](https://github.com/ShelterTechSF/askdarcel-web/blob/71c648d9c2be897b38ee3246d7c4fba0659b2e98/app/styles/utils/_mixins.scss#L4-L6) does some weird stuff to reverse compute what the `em` size should be based on the font-size of the current element, but it normalizes based on `rem` anyway.